docs/templates.md

```diff
@@ -0,0 +1,26 @@
+# Decentralized Application Templates
+
+The templates can be found [here](https://github.com/scale-it/algo-builder-templates) and are extremely helpful and easy to use. They are designed and implemented in such a way that they can provide the aspiring developers a headstart in building dApps based on `Algorand Blockchain`. The templates heavily use [algosdk](https://github.com/algorand/js-algorand-sdk), [algo-builder](https://github.com/scale-it/algo-builder) and [AlgoSigner](https://github.com/PureStake/algosigner).
+
+## Usage
+
+An `algob` project and an `react` frontend are encapsulated in each template. Deployment information is stored in `checkpoints` (in `/artifacts`) from which user can import information directly (loading `.yaml` file in react-app).
+
+The templates can be easily imported or unboxed using the `algob unbox-template` command. 
+
+After successfully unboxing the template, please link the `algob` package in the template directory to use it for running scripts.
+
+
+## Quick Start
+
+The first step is to link `algob` package in the global bin. The documentation for the same can be found [here](https://github.com/scale-it/algo-builder/blob/master/packages/algob/README.md).
```

Comment:
Doesn't necessarily have to be link. User can also add `algob` as a dependency. Linking would be recommended though

Reply:
okay